x-pack/plugin/spatial/src/test/java/org/elasticsearch/xpack/spatial/common/H3CartesianUtilTests.java

```diff
@@ -259,4 +228,18 @@ public void testBoundingBox() {
             }
         }
     }
+
+    public void testWronglyNormaliseGeometry() throws IOException {
+        // this he bin fails normalising in CCW, so we added hack to normalise it CW. If this test fails
```

Comment:
Not sure the word `he` makes sense here?

Reply:
make the test better